lua/telescope/finders.lua

```diff
@@ -115,6 +125,7 @@ function OneshotJobFinder:new(opts)
   local obj = setmetatable({
     fn_command = opts.fn_command,
     entry_maker = opts.entry_maker or make_entry.from_string,
+    fn_preprocess = opts.fn_preprocess,
```

Comment:
@psiska It would be extremely useful for future us if a comment here explain what fn_preporcess does and when to use it. Other than that we should merge

Reply:
Of course I will put there a description. Thanks

Reply:
I've just pushed the documentation. Since I am new to lua, not sure if it is enough. 
Also I've included the section in Readme.md with usage example.